src/engine/UIManager.hxx

```diff
@@ -202,4 +205,12 @@ private:
   bool m_showDebugMenu = false;
 };
 
+struct ImFont;
+struct ImGuiVars {
+
+  static ImFont *buttonFontMainMenu;
+  static ImFont *buttonFontDefault;
+  static void init();
```

Comment:
this config always exist and only one, do we really need direct object for this?

Reply:
Do we need it to exist forever or only during MainMenu?

Reply:
this base structure for imgui config in game, it cant be refactor later

Reply:
I prefer to keep things local if possible. So if it is possible to avoid global structure, I think it is better to do so. So far the global structure is not needed for sure. If it ever shows as a good approach to having a global structure, I am not against it, but not before the time.

Reply:
just with this code, later can change it

Reply:
Sorry, but I do not see it as a valid argument.
So why leave something for later if it can be done now?
I would agree with you if you give any valid reason against my proposal.

Reply:
every time when you will create ctor for temp object json will parse again and added fonts 
better do it once on load